packages/manager/apps/web/client/app/dns-zone/translations/Messages_fr_FR.json

```diff
@@ -12,10 +12,11 @@
   "dashboard_history_popup_title": "Données de la zone DNS",
   "dashboard_history_close_popup": "Fermer",
   "dashboard_history_compare_button_label": "Comparer les versions",
+  "dashboard_history_restoration_successful": "La restauration de votre zone DNS est en cours. La propagation sera effective selon la durée du TTL (maximum 24h). Vous pourrez suivre sa progression dans l'onglet \"taches récentes\".",
```

Comment:
the tab is named "Tâches récentes"

Reply:
Oups, sorry just copy pasted the provided message without checking, my bad